Makefile

```
@@ -0,0 +1,4 @@
+PLATFORM?=linux/amd64
+
+build:
```

possible solution would be to run npm install here
```suggestion
build:
   npm install
```